config/database.yml

```diff
@@ -17,6 +17,8 @@
 default: &default
```

Comment:
@dinakartejat  This file should not be part of the PR, the changes in this file are your local changes.